src/__tests__/__snapshots__/apps.js.snap

```diff
@@ -1162,11 +1176,15 @@ exports[`App Tile renders 1`] = `
       <div
         className="_class-avatar-106"
       >
-        <div>
+        <div
+          onClick={[Function]}
+          onKeyPress={[Function]}
```

Comment:
I hate to be the bearer of bad news, but I think `onKeyPress` is the non-recommended one here D:

https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onkeypress mentions it's deprecated and recommends using onkeydown instead.

Reply:
Ah, good catch.